src/components/BulkActions/BulkActions.tsx

```diff
@@ -465,6 +465,19 @@ class BulkActionsInner extends PureComponent<CombinedProps, State> {
     );
   }
 
+  private isNewBadgeInBadgeActions() {
+    const actions = this.actionSections();
+    if (!actions) return false;
+
+    for (const action of actions) {
+      for (const item of action.items) {
```

Comment:
Why this nested loop vs using something like `.some`?

Reply:
I only copied it over so I'm not sure. However, even using `.some` you would still have to have an inner loop